doc/specs/vulkan/chapters/VK_KHR_swapchain/wsi.txt

```diff
@@ -115,13 +115,14 @@ include::../../api/structs/VkSwapchainCreateInfoKHR.txt[]
     The swapchain is associated with pname:surface.
   * pname:minImageCount is the minimum number of presentable images that the
     application needs.
-    The platform will either create the swapchain with at least that many
-    images, or will fail to create the swapchain.
-  * pname:imageFormat is a slink:VkFormat that is valid for swapchains on
-    the specified surface.
-  * pname:imageColorSpace is a slink:VkColorSpaceKHR that is valid for
-    swapchains on the specified surface.
-  * pname:imageExtent is the non-zero size (in pixels) of the swapchain.
+    The implementation will either create the swapchain with at least that
+    many images, or it will fail to create the swapchain.
+  * pname:imageFormat is a slink:VkFormat that the swapchain image(s) will
+    be created with.
+  * pname:imageColorSpace is a slink:VkColorSpaceKHR that identifies how
+    will the swapchain interpret the swapchain image data.
+  * pname:imageExtent is the non-zero size (in pixels) of the swapchain
```

Comment:
nit: Existing problem, but given that the non-zero-ness is in the valid usage, this part should be removed. It should just say:

> pname:imageExtent is the size (in pixels) of the swapchain image(s).

Reply:
Given that "non-zero" was just added with this week's release, I'd rather not remove it in this pull request.  Probably best to discuss internally.

Reply:
Um, from what I could see after I wrote this there's an internal change in flight to *remove* it... happy to double check though.

Reply:
It is tad excessive though. It is mentioned here, in the VU, and in the Note also.
Then again, it is just one small word. What's the harm in keeping it?

Reply:
If this is contentious, happy to not do it in this PR.

Reply:
I don't care personally, but wanted the author of those words to have a chance to weigh in.  I just sent him/Tobias a private email, pointing him here.

Reply:
It is always good to have a spare Tobias...

Reply:
I'm author of those words. It looks like the non-zeroness is replicated in the three places pointed out: VU, NOTE, and parameter definition.

The VU section is the one place that must state the values cannot be zero.

The .NOTE is meant to be helpful, as the interaction is non-obvious, and multiple shipping applications were found to blow up when the window size became zero. The .NOTE is not essential, although it seems helpful.

If the normative language in the parameter definition does not need to mention zero values, then I agree with removing the non-zero clause from this section, and relying instead on the normative language in the VU section.  I defer to others to explain the requirement on normative language in the parameter section.

Reply:
Generally the parameter descriptions are short and to the point. Anything more verbose goes to the "description" text, and we only usually mention VU stuff in the actual description text if we feel it aids understanding - otherwise it stays exclusively in the VU.

Reply:
Thanks for that clarification. I was overly verbose, and the proposal make sense to remove this "zeroness" language from the parameter description.